.circleci/config.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           command: ./scripts/verify-no-string-interpolation-leftovers
       - run:
           name: Run static code analyzer
-          command: ./gradlew --warn checkstyleMain
+          command: ./gradlew --warn checkstyleMain checkstyleTestFixtures
```

Comment:
what is this option about?

Reply:
To run checkstyle on `testCommon/src/testFixtures/`! https://docs.gradle.org/current/userguide/java_testing.html#sec:java_test_fixtures